translate.js

```diff
@@ -48,6 +49,14 @@ function translateJsonCompilerOutput (output) {
           'sourceMap': contractInput['srcmapRuntime']
         },
         'methodIdentifiers': contractInput['functionHashes'],
+        'gasEstimates': {
+          'creation': {
+            'dataCost': gasEstimates['creation'][1],
+            'executionCost': gasEstimates['creation'][0]
+          },
+          'internal': gasEstimates['internal'],
+          'external': gasEstimates['external']
```

Comment:
as string? that's not what is in http://solidity.readthedocs.io/en/develop/using-the-compiler.html#compiler-input-and-output-json-description